### DISCUSSION TITLE
recipes/lunches/sweet-potato-chickpea-buddha-bowl/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1007] [Comments]
# recipes/lunches/sweet-potato-chickpea-buddha-bowl/

A collection of my recipes.

https://nicholaswilde.io/recipes/lunches/sweet-potato-chickpea-buddha-bowl/

**Comment by nicholaswilde** [comment]
Add the peeling of the sweet potatoes

**Reply by nicholaswilde** [reply]
Add TJs sweet jalapenos and cilantro
